net/validators.go

```diff
@@ -0,0 +1,60 @@
+package net
```

Comment:
I remember seeing this before too.  Is this more rebase weirdness or did this PR get closed down and now these commits are coming in?

If its the second one then please put these commits in another PR as they seem very extractable and orthogonal to the status command.

Reply:
Definitely in #3305, what's up with these changes showing up here too?